atc/gc/worker_collector.go

```diff
@@ -43,6 +46,9 @@ func (wc *workerCollector) Run(ctx context.Context) error {
 
 	if len(affected) > 0 {
 		logger.Info("marked-workers-as-stalled", lager.Data{"count": len(affected), "workers": affected})
+
+		// workers_stalled
```

Comment:
Would it make more sense to have it as `workers_marked_stalled` as that's the actual action taking place? 🤔  

That way it'd make sense to be per-ATC (still being possible to be `sum`med up).